services/pushover/pushovertest/pushovertest.go

```diff
@@ -0,0 +1,85 @@
+package pushovertest
+
+import (
+	"io/ioutil"
+	"net/http"
+	"net/http/httptest"
+	"net/url"
+	"strconv"
+	"sync"
+)
+
+type Server struct {
+	mu       sync.Mutex
+	ts       *httptest.Server
+	URL      string
+	requests []Request
+	closed   bool
+}
+
+func NewServer() *Server {
+	s := new(Server)
+	ts := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		fr := Request{}
+		data, _ := ioutil.ReadAll(r.Body)
+		v, _ := url.ParseQuery(string(data))
+		fr.PostData, _ = NewPostData(v)
+		s.mu.Lock()
+		s.requests = append(s.requests, fr)
+		s.mu.Unlock()
+	}))
+	s.ts = ts
+	s.URL = ts.URL
+	return s
+}
+
+func (s *Server) Requests() []Request {
+	s.mu.Lock()
+	defer s.mu.Unlock()
+	return s.requests
+}
+func (s *Server) Close() {
+	if s.closed {
+		return
+	}
+	s.closed = true
+	s.ts.Close()
+}
+
+type Request struct {
+	PostData PostData
+}
+
+type PostData struct {
+	Token     string `json:"token"`
+	User      string `json:"user"`
+	Message   string `json:"message"`
+	Device    string `json:"device"`
+	Title     string `json:"title"`
+	URL       string `json:"url"`
+	URLTitle  string `json:"url_title"`
+	Sound     string `json:"sound"`
+	Timestamp string `json:"timestamp"`
+	Priority  int    `json:"priority"`
+}
```

Comment:
Whoops. My mistake. This is left-over from when I thought the Pushover API accepted JSON.